### ISSUE TITLE
sendToWebview() not working

### ISSUE THREAD

**Issue by der-lukas** [skpm/sketch-module-web-view#89]
Hey!

I'm trying to trigger a function in the webview from within an "SelectionChanged"-Action function and tried to use this exact example code you supplied in the docs:

```
import { isWebviewPresent, sendToWebview } from 'sketch-module-web-view/remote'

if (isWebviewPresent('unique.id')) {
  sendToWebview('unique.id', 'someGlobalFunctionDefinedInTheWebview("hello")')
}
```

but I can't get it to work and always get the following error message:

<img width="829" alt="bildschirmfoto 2019-03-03 um 00 54 51" src="https://user-images.githubusercontent.com/5270044/53689160-9dc2a280-3d4f-11e9-99b9-904acc85b225.png">

This is the code I have in `selection-changed.js`:

```
if (isWebviewPresent('smartStyleWindow')) {
  sendToWebview('smartStyleWindow', 'someGlobalFunctionDefinedInTheWebview("hello")');
 } else {
   console.log("Window not active!");
}
```

and this is the function inside `webview.js`, which I imported in `webview.html`:

```
window.someGlobalFunctionDefinedInTheWebview = function (arg) {
    console.log(arg);
}
```

Any help is very appreciated!

**Comment by mathieudutour** [comment]
should be fixed in 2.1.7

**Comment by so77id** [comment]
this problem was solved? i am experiencing this problem now
